lib/src/RadioButton.js

```diff
@@ -66,20 +72,26 @@ export default class RadioButton extends Component {
           style={[
             outerStyle(size, outerColor),
             transformStyle(springValue),
-            styles.center
+            styles.center,
           ]}
         >
           {stateManagement ? (
             <View style={innerStyle(size, innerColor)} {...this.props} />
           ) : null}
         </Animated.View>
-        {disableText || (
-          <View style={textContainer(horizontal)}>
+        <View style={textContainer(horizontal)}>
+          {!disableImage && (
+            <ImageComponent
+              style={imageStyle(width, height)}
```

Comment:
We can remove both `width` and `height` props. You can simply create your own imageStyle from style file and let the `imageStyle` prop from the developer.